galaxy.yml

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 ---
-version: 1.0.10
+version: 1.2.2
```

Comment:
This isn't an issue that needs addressed but for future releases, you can use '-dev' as a suffix to your versions in Automation Hub so that we keep the versions in GitHub and Automation Hub in sync. 

Example: You can test with version 1.2.0-dev1, 1.2.0-dev2, and so on and then create version 1.2.0 after the code has been merged.